sdks/js/keelson/payloads/index.google.protobuf.ts

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable */
```

Comment:
Are these also generated files? I.e. the ones that start with `index.*`

Reply:
yes, they are generated because of this flag to ts-proto
    --ts_proto_opt=outputIndex=true \